core/src/main/java/com/linecorp/armeria/common/logging/RequestLogBuilder.java

```diff
@@ -40,11 +40,6 @@
  */
 public interface RequestLogBuilder {
 
-    /**
-     * A dummy {@link RequestLogBuilder} that discards everything it collected.
-     */
-    RequestLogBuilder NOOP = new NoopRequestLogBuilder();
```

Comment:
Wonder if we need to mention this in the release note. Probably there's no one who has used this. 🤣

Reply:
Maybe `@Deprecate` first? :sob:

Reply:
Oops, Sorry. I will revert it and `@Deprecated` it.

Reply:
Fix it https://github.com/line/armeria/pull/2248/commits/79fd34ef6172333f41855ad49a7783d75723a2d1
Wrote documentation for the deprecation. But I am wondering if it is a good explanation. 😅